
        
12:33:04 PM: DEBUG [logger_base.py:12] Mensaje a nivel Debug
12:33:04 PM: INFO [logger_base.py:13] Mensaje a nivel de info
12:33:04 PM: WARNING [logger_base.py:14] Mensaje a nivel de warning
12:33:04 PM: ERROR [logger_base.py:15] Mensaje a nivel de error
12:33:04 PM: CRITICAL [logger_base.py:16] Mensaje a nivel de crítico
12:33:07 PM: DEBUG [Persona.py:53] 
            Id Persona: 1, Nombre: Juan,
            Apellido: Pérez, Email: jperez@mail.com
        
12:33:07 PM: DEBUG [Persona.py:57] 
            Id Persona: None, Nombre: Juan,
            Apellido: Pérez, Email: jperez@mail.com
        
12:33:07 PM: DEBUG [Persona.py:61] 
            Id Persona: 1, Nombre: None,
            Apellido: None, Email: None
        
12:46:04 PM: DEBUG [conexion.py:24] Conexión exitosa: <connection object at 0x000001ED887957B0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
12:47:47 PM: DEBUG [conexion.py:24] Conexión exitosa: <connection object at 0x00000204EEAE67B0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
12:47:47 PM: DEBUG [conexion.py:37] Se abrió correctamente el cursor: <cursor object at 0x00000204F07D1AC0; closed: 0>
12:57:35 PM: DEBUG [conexion.py:24] Conexión exitosa: <connection object at 0x0000025ED2E157B0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
12:57:35 PM: DEBUG [conexion.py:37] Se abrió correctamente el cursor: <cursor object at 0x0000025ED4B3D9E0; closed: 0>
12:57:35 PM: DEBUG [persona_dao.py:32] 
            Id Persona: 1, Nombre: Juan,
            Apellido: Perez, Email: jperez@mail.com
        
12:57:35 PM: DEBUG [persona_dao.py:32] 
            Id Persona: 2, Nombre: Juan Carlos,
            Apellido: Juarez, Email: jcjuarez@mail.com
        
12:57:35 PM: DEBUG [persona_dao.py:32] 
            Id Persona: 3, Nombre: Carlos,
            Apellido: Lara, Email: clara@mail.com
        
01:04:26 PM: DEBUG [conexion.py:24] Conexión exitosa: <connection object at 0x000002364E3867B0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
01:05:51 PM: DEBUG [conexion.py:24] Conexión exitosa: <connection object at 0x000001E7761F67B0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
01:05:51 PM: DEBUG [persona_dao.py:34] Persona insertada: 
            Id Persona: None, Nombre: Pepito,
            Apellido: Grillo, Email: p@mail.com
        
01:05:51 PM: DEBUG [persona_dao.py:41] Personas insertadas: 1
01:05:51 PM: DEBUG [conexion.py:37] Se abrió correctamente el cursor: <cursor object at 0x000001E777F14AC0; closed: 0>
01:05:51 PM: DEBUG [persona_dao.py:45] 
            Id Persona: 1, Nombre: Juan,
            Apellido: Perez, Email: jperez@mail.com
        
01:05:51 PM: DEBUG [persona_dao.py:45] 
            Id Persona: 2, Nombre: Juan Carlos,
            Apellido: Juarez, Email: jcjuarez@mail.com
        
01:05:51 PM: DEBUG [persona_dao.py:45] 
            Id Persona: 3, Nombre: Carlos,
            Apellido: Lara, Email: clara@mail.com
        
01:05:51 PM: DEBUG [persona_dao.py:45] 
            Id Persona: 7, Nombre: Pepito,
            Apellido: Grillo, Email: p@mail.com
        
01:14:09 PM: DEBUG [conexion.py:24] Conexión exitosa: <connection object at 0x00000205A38167B0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
01:14:09 PM: DEBUG [conexion.py:37] Se abrió correctamente el cursor: <cursor object at 0x00000205A5526AC0; closed: 0>
01:14:09 PM: DEBUG [persona_dao.py:44] Persona actualizada: 
            Id Persona: 1, Nombre: Pepito,
            Apellido: Grillo, Email: p@mail.com
        
01:14:09 PM: DEBUG [persona_dao.py:56] Personas actualizadas: 1
01:21:16 PM: DEBUG [conexion.py:24] Conexión exitosa: <connection object at 0x000001C2D96A67B0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
01:21:16 PM: DEBUG [conexion.py:37] Se abrió correctamente el cursor: <cursor object at 0x000001C2DB3B6BA0; closed: 0>
01:21:16 PM: DEBUG [persona_dao.py:45] Persona actualizada: 
            Id Persona: 1, Nombre: Pepito,
            Apellido: Grillo, Email: p@mail.com
        
01:21:16 PM: DEBUG [persona_dao.py:57] Personas actualizadas: 1
01:21:48 PM: DEBUG [conexion.py:24] Conexión exitosa: <connection object at 0x00000189282B67B0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
01:21:48 PM: DEBUG [conexion.py:37] Se abrió correctamente el cursor: <cursor object at 0x0000018929F96BA0; closed: 0>
01:21:48 PM: DEBUG [persona_dao.py:45] Persona actualizada: 
            Id Persona: 1, Nombre: Pepito,
            Apellido: Grillo, Email: p@mail.com
        
01:21:48 PM: DEBUG [persona_dao.py:57] Personas actualizadas: 1
01:21:48 PM: DEBUG [persona_dao.py:61] 
            Id Persona: 1, Nombre: Pepito,
            Apellido: Grillo, Email: p@mail.com
        
01:21:48 PM: DEBUG [persona_dao.py:61] 
            Id Persona: 2, Nombre: Juan Carlos,
            Apellido: Juarez, Email: jcjuarez@mail.com
        
01:21:48 PM: DEBUG [persona_dao.py:61] 
            Id Persona: 3, Nombre: Carlos,
            Apellido: Lara, Email: clara@mail.com
        
01:21:48 PM: DEBUG [persona_dao.py:61] 
            Id Persona: 7, Nombre: Pepito,
            Apellido: Grillo, Email: p@mail.com
        
01:25:44 PM: DEBUG [conexion.py:24] Conexión exitosa: <connection object at 0x0000021DB75057B0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
01:25:44 PM: DEBUG [conexion.py:37] Se abrió correctamente el cursor: <cursor object at 0x0000021DB91F6BA0; closed: 0>
01:25:44 PM: DEBUG [persona_dao.py:55] Objeto eliminado: 
            Id Persona: 20, Nombre: None,
            Apellido: None, Email: None
        
01:25:44 PM: DEBUG [persona_dao.py:71] Personas eliminadas: 0
01:25:44 PM: DEBUG [persona_dao.py:75] 
            Id Persona: 1, Nombre: Pepito,
            Apellido: Grillo, Email: p@mail.com
        
01:25:44 PM: DEBUG [persona_dao.py:75] 
            Id Persona: 2, Nombre: Juan Carlos,
            Apellido: Juarez, Email: jcjuarez@mail.com
        
01:25:44 PM: DEBUG [persona_dao.py:75] 
            Id Persona: 3, Nombre: Carlos,
            Apellido: Lara, Email: clara@mail.com
        
01:25:44 PM: DEBUG [persona_dao.py:75] 
            Id Persona: 7, Nombre: Pepito,
            Apellido: Grillo, Email: p@mail.com
        
01:36:34 PM: DEBUG [conexion.py:24] Conexión exitosa: <connection object at 0x000002352F4267B0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
01:36:34 PM: DEBUG [conexion.py:37] Se abrió correctamente el cursor: <cursor object at 0x0000023531126BA0; closed: 0>
01:36:34 PM: DEBUG [persona_dao.py:55] Objeto eliminado: 
            Id Persona: 7, Nombre: None,
            Apellido: None, Email: None
        
01:36:34 PM: DEBUG [persona_dao.py:71] Personas eliminadas: 1
01:36:34 PM: DEBUG [persona_dao.py:75] 
            Id Persona: 1, Nombre: Pepito,
            Apellido: Grillo, Email: p@mail.com
        
01:36:34 PM: DEBUG [persona_dao.py:75] 
            Id Persona: 2, Nombre: Juan Carlos,
            Apellido: Juarez, Email: jcjuarez@mail.com
        
01:36:34 PM: DEBUG [persona_dao.py:75] 
            Id Persona: 3, Nombre: Carlos,
            Apellido: Lara, Email: clara@mail.com
        
11:54:27 AM: DEBUG [pool_conexion.py:26] Creación del pool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x000001F0DC971FA0>
11:54:27 AM: DEBUG [pool_conexion.py:37] Conexión obtenida del pool: <connection object at 0x000001F0DC9AFE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
11:54:27 AM: DEBUG [pool_conexion.py:37] Conexión obtenida del pool: <connection object at 0x000001F0DE4C67B0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
11:56:38 AM: DEBUG [pool_conexion.py:26] Creación del pool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x0000016EB0B64FA0>
11:56:38 AM: DEBUG [pool_conexion.py:37] Conexión obtenida del pool: <connection object at 0x0000016EB0BA0E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
11:56:38 AM: DEBUG [pool_conexion.py:37] Conexión obtenida del pool: <connection object at 0x0000016EB26C67B0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
11:56:38 AM: DEBUG [pool_conexion.py:37] Conexión obtenida del pool: <connection object at 0x0000016EB26C6150; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
11:56:38 AM: DEBUG [pool_conexion.py:37] Conexión obtenida del pool: <connection object at 0x0000016EB26C6AE0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
11:56:38 AM: DEBUG [pool_conexion.py:37] Conexión obtenida del pool: <connection object at 0x0000016EB26C69D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
12:02:39 PM: DEBUG [pool_conexion.py:26] Creación del pool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x00000150A3844F40>
12:02:39 PM: DEBUG [pool_conexion.py:37] Conexión obtenida del pool: <connection object at 0x00000150A3881E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
12:02:39 PM: DEBUG [pool_conexion.py:43] Regresamos la conexión al pool: <connection object at 0x00000150A3881E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
12:02:39 PM: DEBUG [pool_conexion.py:37] Conexión obtenida del pool: <connection object at 0x00000150A3881E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
12:02:39 PM: DEBUG [pool_conexion.py:37] Conexión obtenida del pool: <connection object at 0x00000150A53A77B0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
12:02:39 PM: DEBUG [pool_conexion.py:37] Conexión obtenida del pool: <connection object at 0x00000150A53A7150; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
12:02:39 PM: DEBUG [pool_conexion.py:37] Conexión obtenida del pool: <connection object at 0x00000150A53A7AE0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
12:02:39 PM: DEBUG [pool_conexion.py:37] Conexión obtenida del pool: <connection object at 0x00000150A53A79D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
12:23:02 PM: DEBUG [cursor_del_pool.py:11] Inicio del método with __nenter__
12:23:02 PM: DEBUG [pool_conexion.py:26] Creación del pool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x0000025A29963EB0>
12:23:02 PM: DEBUG [pool_conexion.py:37] Conexión obtenida del pool: <connection object at 0x0000025A299BEE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
12:23:02 PM: DEBUG [cursor_del_pool.py:30] Dentro del bloque with
12:23:02 PM: DEBUG [cursor_del_pool.py:32] [(3, 'Carlos', 'Lara', 'clara@mail.com'), (2, 'Juan Carlos', 'Juarez', 'jcjuarez@mail.com'), (1, 'Pepito', 'Grillo', 'p@mail.com')]
12:23:02 PM: DEBUG [cursor_del_pool.py:17] Se ejecuta método __exit__
12:23:02 PM: DEBUG [cursor_del_pool.py:23] Commit de la transacción
12:23:02 PM: DEBUG [pool_conexion.py:43] Regresamos la conexión al pool: <connection object at 0x0000025A299BEE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
12:23:57 PM: DEBUG [cursor_del_pool.py:11] Inicio del método with __nenter__
12:23:57 PM: DEBUG [pool_conexion.py:26] Creación del pool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x0000024F13D43EB0>
12:23:57 PM: DEBUG [pool_conexion.py:37] Conexión obtenida del pool: <connection object at 0x0000024F13D86E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
12:23:57 PM: DEBUG [cursor_del_pool.py:30] Dentro del bloque with
12:23:57 PM: DEBUG [cursor_del_pool.py:17] Se ejecuta método __exit__
12:23:57 PM: ERROR [cursor_del_pool.py:20] Ocurrió una excepción, hace rollback: no existe la relación «person»
LINE 1: SELECT * FROM person
                      ^
 <class 'psycopg2.errors.UndefinedTable'> <traceback object at 0x0000024F158DC940>
12:23:57 PM: DEBUG [pool_conexion.py:43] Regresamos la conexión al pool: <connection object at 0x0000024F13D86E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
12:30:41 PM: DEBUG [cursor_del_pool.py:11] Inicio del método with __enter__
12:30:41 PM: DEBUG [pool_conexion.py:26] Creación del pool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x000001E08B1349D0>
12:30:41 PM: DEBUG [pool_conexion.py:37] Conexión obtenida del pool: <connection object at 0x000001E0895E0E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
12:30:41 PM: DEBUG [cursor_del_pool.py:17] Se ejecuta método __exit__
12:30:41 PM: DEBUG [cursor_del_pool.py:23] Commit de la transacción
12:30:41 PM: DEBUG [pool_conexion.py:43] Regresamos la conexión al pool: <connection object at 0x000001E0895E0E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
12:30:41 PM: DEBUG [persona_dao_pool.py:72] 
            Id Persona: 1, Nombre: Pepito,
            Apellido: Grillo, Email: p@mail.com
        
12:30:41 PM: DEBUG [persona_dao_pool.py:72] 
            Id Persona: 2, Nombre: Juan Carlos,
            Apellido: Juarez, Email: jcjuarez@mail.com
        
12:30:41 PM: DEBUG [persona_dao_pool.py:72] 
            Id Persona: 3, Nombre: Carlos,
            Apellido: Lara, Email: clara@mail.com
        
